otter/cloud_client/__init__.py

```diff
@@ -705,21 +708,45 @@ def get_clbs():
         success=lambda (response, body): body['loadBalancers'])
 
 
-def get_clb_node_feed(lb_id, node_id):
-    """Get the atom feed associated with a CLB node. Returns feed as str."""
+def _node_feed_request(lb_id, node_id, params):
     return service_request(
-        ServiceType.CLOUD_LOAD_BALANCERS,
-        'GET',
+        ServiceType.CLOUD_LOAD_BALANCERS, 'GET',
         append_segments('loadbalancers', str(lb_id), 'nodes',
                         '{}.atom'.format(node_id)),
+        params=params,
         json_response=False
     ).on(
         error=_only_json_api_errors(
             lambda c, b: _process_clb_api_error(c, b, lb_id))
     ).on(
         log_success_response('request-get-clb-node-feed', identity)
-    ).on(
-        success=lambda (response, body): body)
+    ).on(itemgetter(1))
+
+
+@do
+def get_clb_node_feed(lb_id, node_id):
+    """
+    Get the atom feed associated with a CLB node.
+
+    :param int lb_id: Cloud Load balancer ID
+    :param int node_id: Node ID of in loadbalancer node
+
+    :returns: ``list`` of atom entry :class:`Element`
```

Comment:
> Is this accurate?

No.

> Doesn't a @do-decorated function always return an Effect?

Yes.

Thanks for pointing this out. I start thinking in terms of returning value wrapped in the Effect when I am working with a lot of effect code which results in above docstring similar to how it happens when working with a lot of `Deferred` code :). I'll change it.